silkworm/rpc/commands/eth_api.cpp

```diff
@@ -910,7 +910,7 @@ Task<void> EthereumRpcApi::handle_eth_estimate_gas(const nlohmann::json& request
         reply = make_json_error(request, 100, error_msg);
         co_return;
     }
-    const auto call = params[0].get<Call>();
+    auto call = params[0].get<Call>();
```

Comment:
I would prefer to keep this `const` as it is currently. I've seen why you're changing it below, but I think you can achieve the same goal with a different change. More on it in the comments below.

Reply:
ok